### PR TITLE
FTheoryTools: Use @vprint and add README.md

### LINKED ISSUE #2180: A lot of infos during tests about "AUXILIARY ambient space"
There are quite are lot of messages of the form 
```
[ Info: Base space was not fully specified. Returning AUXILIARY ambient space.
```
during the tests. Is there a way to disable this? See https://github.com/oscar-system/Oscar.jl/actions/runs/4577059265/jobs/8082008614#step:6:1194

Seem to be coming from the FTheoryTools. CC: @HereAround

### PR DESCRIPTION
**Pull request by HereAround**
This PR has two goal:
1. Fix https://github.com/oscar-system/Oscar.jl/issues/2180.
2. Add file `README.md` in Oscar.jl/experimental/FTheoryTools to comply with the new structure for experimental code.

cc @apturner

### CODE REVIEW

File: experimental/FTheoryTools/src/TateModels/attributes.jl [hunk 1]
```diff
@@ -114,7 +114,7 @@ Normal toric variety without torusfactor
 ```
 """
 @attr AbstractNormalToricVariety function toric_base_space(t::GlobalTateModel)
-    base_fully_specified(t) || @info("Base space was not fully specified. Returning AUXILIARY base space.")
+    base_fully_specified(t) || @vprint :GlobalTateModel 1 "Base space was not fully specified. Returning AUXILIARY base space."
```
Comment (fingolfin) [inline experimental/FTheoryTools/src/TateModels/attributes.jl]: I think you may also need to use `add_verbosity_scope(:GlobalTateModel)` as described in the `@vprint` docstring?

Comment (HereAround) [reply experimental/FTheoryTools/src/TateModels/attributes.jl]: Got it. Thank you!

File: src/Oscar.jl [hunk 2]
```diff
@@ -249,6 +249,10 @@ function __init__()
 
     add_verbose_scope(:K3Auto)
     add_assert_scope(:K3Auto)
+    
+    add_verbose_scope(:GlobalTateModel)
+    add_verbose_scope(:GlobalWeierstrassModel)
```
Comment (HereAround) [inline src/Oscar.jl:254]: @thofma Initially, I tried to add these lines at the very top of `Oscar.jl/experimental/FTheoryTools/src/FTheoryTools.jl`. This is the top-level src-file of `FTheoryTools` which includes all other files. So this seemed a natural place, given that `FTheoryTools` is experimental.

Still, if added at the very top of `Oscar.jl/experimental/FTheoryTools/src/FTheoryTools.jl`, I received an `ErrorException` showing the error message "Not a valid symbol" when running the tests. Why is that?

Comment (thofma) [reply src/Oscar.jl:254]: They need to be added very early in the loading of a package, so must go into `__init__`. If `FTheoryTools` would have its own module, you could put it in the `__ini__` function of that module, like here:

https://github.com/oscar-system/Oscar.jl/blob/915aeb9de5c3ffe97ffcbeca20b6fbca8e61d22c/experimental/GModule/Cohomology.jl#L9-L15

But just putting those in the `__init__` of Oscar itself is also fine.

Comment (HereAround) [reply src/Oscar.jl:254]: Thank you. That makes sense.

File: experimental/FTheoryTools/README.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+## Contact
+
+Please direct questions about this part of OSCAR to the following people:
+* [Martin Bies](https://martinbies.github.io/),
+* [Andrew Turner](https://apturner.net/).
+
+You can ask questions in the [OSCAR Slack](https://www.oscar-system.org/community/#slack).
+
+Alternatively, you can [raise an issue on github](https://www.oscar-system.org/community/#how-to-report-issues).
```
Comment (fingolfin) [inline experimental/FTheoryTools/README.md]:
```suggestion
Alternatively, you can [raise an issue on GitHub](https://github.com/oscar-system/Oscar.jl).
```

**Review by fingolfin — APPROVED** [review]

**Review by thofma — APPROVED** [review]
Looks, good thanks!

**Pull request merged**